src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -302,6 +300,14 @@
 		7567A2102DA5ABCF00D91529 /* DefaultServerCluster.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A20B2DA5ABCE00D91529 /* DefaultServerCluster.cpp */; };
 		7567A2112DA5ABCF00D91529 /* ServerClusterInterface.h in Headers */ = {isa = PBXBuildFile; fileRef = 7567A20C2DA5ABCE00D91529 /* ServerClusterInterface.h */; };
 		7567A2122DA5ABCF00D91529 /* ServerClusterInterface.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A20D2DA5ABCE00D91529 /* ServerClusterInterface.cpp */; };
+		757DA30A2DB035A900E4AD75 /* DefaultServerCluster.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A20B2DA5ABCE00D91529 /* DefaultServerCluster.cpp */; };
```

Comment:
I don't quite understand this change.  We already compiled in DefaultServerCluster.cpp (see line three lines above).  What is this actually adding `DefaultServerCluster.cpp` and `DefaultServerCluster.h` to?

Reply:
I used the XCode IDE here ... I have no idea what this contains and why. There seem to be more duplications in other places.

I have no good way to edit pbxproj files.

Reply:
You used the IDE to do what, though?  Did you change anything about the DefaultServerCluster bits in the UI there at all?

Reply:
I added a group and added files to it. I am not as familiar with XCode though ...
I used the test of "does this pass CI/still compiles" and if that passed it seemed ok to me.